src/herder/HerderImpl.cpp

```diff
@@ -861,7 +806,8 @@ HerderImpl::getJsonInfo(size_t limit)
         mApp.getConfig().toStrKey(mApp.getConfig().NODE_SEED.getPublicKey());
 
     ret["scp"] = getSCP().getJsonInfo(limit);
-    ret["queue"] = mPendingEnvelopes.getJsonInfo(limit);
+    ret["queue"] = mApp.getOverlayManager().getJsonInfo(limit);
```

Comment:
let's not use the same `queue` element as it's not managed by herder anymore.
this also means that we should not invoke overlay status from `herder` but instead have `CommandHandler::scpInfo` call the relevant overlay method